R/config.R

```diff
@@ -418,7 +421,9 @@ check_resources <- function(cluster, template, cores, wholenode, parallel) {
       stop("Cannot specify both wholenode and cores")
     }
     assert_scalar_integer(cores)
-    max_cores <- if (cluster == "fi--didemrchnb") 64 else 24
+    max_cores <- 24
+    if (cluster == "fi--didemrchnb") max_cores <- 64
+    else if (cluster == "wpia-hpc-hn") max_cores <- 32
```

Comment:
can you move this into a free function?

```r
didehpc_max_cores <- function(cluster) {
 switch(cluster,
        "fi--dideclusthn" = 24
        "fi--didemrchnb" = 64,
        "wpia-hpc-hn" = 32,
        stop(sprintf("Invalid cluster '%s'", cluster))
}
```

and use as

```r
max_cores <- didehpc_max_cores(cluster)
```

or put the check in there too?